### DISCUSSION TITLE
childruntime fwrite broken pipe errno=32

### DISCUSSION THREAD

**Discussion by malohtie** [spatie/async#145] [Q&A]
Hello,
today our server start logging this error 
![image](https://user-images.githubusercontent.com/20033279/116011007-1a8cf680-a612-11eb-9379-218f9a9f17c2.png)
and we dont know if is it a bug in async lib or we're doing something wrong
any idea guys ?

**Comment by kirkbushell** [comment]
I've just come across this and also logged it here: https://github.com/spatie/async/issues/210

There's definitely something funky going on there.
